opm/material/fluidsystems/ThreeComponentFluidSystem.hh

```diff
@@ -210,10 +209,41 @@ namespace Opm {
             assert(phaseIdx < numPhases);
             assert(compIdx < numComponents);
 
-            LhsEval phi = PengRobinsonMixture::computeFugacityCoefficient(fluidState, paramCache, phaseIdx, compIdx);
+            const auto phi = decay<LhsEval>(PengRobinsonMixture::computeFugacityCoefficient(fluidState, paramCache, phaseIdx, compIdx));
             return phi;
```

Comment:
If you're changing this anyway, why not just write it as
```
return decay<LhsEval>(...);
```
instead?

Reply:
same with below.